scss/components/_components.buttons.scss

```diff
@@ -165,9 +165,43 @@
 // Group buttons together and space them accordingly
 
 .c-button-bar {
+	.c-button-group,
 	.c-button {
+		margin-right: 6px;
+
+		&:last-child {
+			margin-right: 0;
+		}
+	}
+}
+
+
+
+// ## Button Group
+// -----------------------------------------------------------------------------
+
+// Merge a group of buttons together into a single unit
+
+.c-button-group {
+	display: flex;
+	flex-wrap: nowrap;
+
+	.c-button {
+		margin-right: 0;
+		border-radius: 0;
+
 		&:not(:last-child) {
-			margin-right: 6px;
```

Comment:
Split this out into:

```scss
.c-button {
    margin-right: 6px;

    &:last-child {
        margin-right: 0;
    }
}
```

Since this uses less specific selectors. This was necessary to easily override the `margin-right` in `.c-button-group`.